js/kucoinfutures.js

```diff
@@ -286,6 +286,10 @@ module.exports = class kucoinfutures extends kucoin {
                     'ERC20': 'eth',
                     'TRC20': 'trx',
                 },
+                'code': 'BTC',
+                'fetchBalance': {
```

Comment:
Apparently, this `fetchBalance` code override is not involved in `fetchBalance`, so maybe we should remove it, or account for it.